git-annex-remote-googledrive

```diff
@@ -153,8 +153,9 @@ class GoogleRemote(ExportRemote):
     def initremote(self):
         self._send_version()
         self.prefix = self.annex.getconfig('prefix')
-        if self.prefix == '' or self.prefix == '/':
-            raise RemoteError("prefix must not be empty.")
+        self.root_id = None
```

Comment:
`root_id` is already used for the ID of the folder to which `prefix` is pointing. It's not a good idea to re-use it for a different purpose. If we did, `git annex enableremote` would create a new prefix folder inside the old root every time it is run.

Reply:
Maybe we could just allow the `prefix` to be empty if `root_id` is set. Then the user can directly pass the ID of the folder that should be used by git-annex. I think this would reduce confusion about where the root folder is going to be created. Users could also use this method in order to make the folder independent from the path. A folder given by `root_id` can be moved around and will still be found by git-annex. What do you think?